.github/workflows/frontend-build-and-publish-apps.yml

```
@@ -56,6 +56,39 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check-version-bump:
+    runs-on: ubuntu-latest
+    if: ${{ github.base_ref == 'master' }}
+
+    steps:
+    - name: Checkout current code
+      uses: actions/checkout@v4
+
+    - name: Get the current version
+      id: current
+      run: echo "version=$(node -e "console.log(require('./package.json').version)")" >> "$GITHUB_OUTPUT"
```

```suggestion
      run: echo "version=$(node -p "require('./package.json').version")" >> "$GITHUB_OUTPUT"
```
should work, but please check

---

yep it works! Thanks :)